glossary/index.md

```diff
@@ -164,6 +164,16 @@ The JSON-formatted genesis configuration file that contains the data necessary t
 ## gigaton
 The denomination of Autonity's [auton](/glossary/#auton) cryptocurrency used to denominate Autonity gas prices. 1 `gigaton` = 1,000,000,000 [ton](/glossary/#ton). The Autonity equivalent of wei is [ton](/glossary/#ton) and of gwei gigaton.
 
+## inflation mechanism
+The mechanism by which a percentage of the total [newton](/glossary/#newton) supply is held in an "Inflation Reserve" and released over time as [inflation awards](/glossary/#inflation-awards).
+
+Newton total supply is capped at 100 million NTNs of which 40% is reserved for inflation awards in the short-to-medium term.
+
+Inflation awards are emitted in the form of newton inflation (i.e. newly minted newton supply) for bonded stake [participating](/glossary/#participation-rate) in securing the [Autonity network](/glossary/#autonity-network). The quantity of newton emissions declines (in average) over time and asymptotically approaches zero as the Inflation Reserve is fixed.
+
+## inflation awards
+Inflation awards are a distribution of newly minted newton by the [inflation mechanism](/glossary/#inflation-mechanism) to all holders of bonded stake [participating](/glossary/#participation-rate) in securing the network.
+
```

Comment:
Should we add a comment on the minted NTNs during the transition period? 

If we denote by $N$ the circulating amount, one could note, during the transition period, that $dN$, the increase in minted supply is greater than the inflation reward amount $i \cdot dt \cdot N$, where $dt$ is the epoch time in years. Indeed, in addition to the inflation reward, the circulating amount is increased by the amount of the unlocked token (for Locked Non-Stakable vesting), thus $$dN = i \cdot dt \cdot N + |\Delta L|$$, where \Delta L| is the unlocked amount during that epoch. 

Otherwise, some will not understand why $dN$ increase is greater than the distributed inflation reward amount.